### ISSUE TITLE
[BUG] No tattoo

### ISSUE THREAD

**Issue by SaraLef** [ParadoxGamingRP/bugs-and-suggestions#1107]
### Your Discord Name

ssara#7821

### A clear and concise description of what the bug is

It's your favourite bugreporter here, reporting to you LIVE from Los Santos! 
We have currently lost all tattoos and there seems to be no one found with any tattos at the moment. 

### Steps to Reproduce the Bug

Wake up from your beauty sleep
Realizing God thought your beauty sleep ment no tattoos
See that he removed them and replaced it with absolutely nothing 
Because God believes in natural beauty.

In short terms
There's no tattoos on anyone rn.

### Expected Behavior

To have my tattoos still.

### Screenshots/Videos

N/A
